Updated PyPI setup access in prerequisites checklist of release how-to. (#16967)
django/django: docs/internals/howto-release-django.txt

* Security releases: disclosing and fixing a vulnerability. This'll
  generally involve two or three simultaneous releases -- e.g.

>   3.2.x, 4.2.x, and, depending on timing, perhaps a 4.1.x.

Reviewer: I don't see much value in changing it. Also, updating to `4.2.x` makes the rest of the sentence _"perhaps a 4.1.x"_ incorrect.

* A GPG key. If the key you want to use is not your default signing key, you'll
  need to add ``-u you@example.com`` to every GPG signing command below, where
  ``you@example.com`` is the email address associated with the key you want to
  use. You will also need to add ``-i you@example.com`` to the ``twine`` call.

* An install of some required Python packages:

  .. code-block:: shell

      $ python -m pip install wheel twine

* Access to Django's project on PyPI. Create a project-scoped token following
  the `official documentation <https://pypi.org/help/#apitoken>`_ and set up
  your ``$HOME/.pypirc`` file like this:

  .. code-block:: ini
     :caption: ``~/.pypirc``

     [distutils]
       index-servers =
         pypi
         django

     [pypi]
       username = __token__

>        password = # user-scoped or project-scoped token, to set as the default

Reviewer: ```suggestion
       password = # User-scoped or project-scoped token, to set as the default.
```

     [django]
       repository = https://upload.pypi.org/legacy/
       username = __token__

>        password = # a project token

Reviewer: ```suggestion
       password = # A project token.
```

* A GPG key. If the key you want to use is not your default signing key, you'll
  need to add ``-u you@example.com`` to every GPG signing command below, where
  ``you@example.com`` is the email address associated with the key you want to
  use. You will also need to add ``-i you@example.com`` to the ``twine``

>   shell invocation.

Reviewer: `-u` is the correct option to use a non-default GPG key, and it's already documented. Please revert.

>   use. You will also need to add ``-i you@example.com`` to the ``twine``
>   shell invocation.

Reviewer: ```suggestion
  use. You will also need to add ``-i you@example.com`` to the ``twine`` call.
```

* An install of some required Python packages:

  .. code-block:: shell

      $ python -m pip install wheel twine

* Access to Django's project on PyPI. Create a project-scoped token following
  the `official documentation <https://pypi.org/help/#apitoken>`_. At the time
  of this writing, that means:

  #. Create a new token in your

>      `account settings page <https://pypi.org/manage/account/token/>`_.

Reviewer: This step is already described in _"Create a project-scoped token following the official documentation"_. I think we can remove the list of steps and simplify it, e.g.
```
* Access to Django's project on PyPI. Create a project-scoped token following
  the `official documentation <https://pypi.org/help/#apitoken>`_ and set up
  your ``$HOME/.pypirc`` file like this:

    .. code-block:: ini
        :caption: ``~/.pypirc``
        ...
```

  #. Set up your ``$HOME/.pypirc`` file like this:

     .. code-block:: ini
        :caption: ``~/.pypirc``

        [distutils]
          index-servers =
            pypi
            PROJECT_NAME

        [pypi]
          username = __token__
          password = # either a user-scoped token or a project-scoped token you want to set as the default

        [PROJECT_NAME]
          repository = https://upload.pypi.org/legacy/
          username = __token__

>           password = # a project token 

Reviewer: This an instruction for us not for other projects, so I'd change `PROJECT_NAME` to `django`.
